pom.xml

```diff
@@ -51,7 +51,7 @@
         <findbugs.version>3.0.0</findbugs.version>
         <geronimo-jms.version>1.0-alpha-2</geronimo-jms.version>
         <glassfish-json.version>1.0.2</glassfish-json.version>
-        <guava.version>19.0</guava.version>
+        <guava.version>29.0-jre</guava.version>
```

Comment:
This doesn't match the version in the release note.

Reply:
fixed